app/src/main/res/layout/activity_library.xml

```diff
@@ -1,9 +1,77 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".LibraryActivity">
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <androidx.cardview.widget.CardView
+        android:id="@+id/CardViewID"
```

Comment:
id's are normally lowercase and use [snake case](https://en.wikipedia.org/wiki/Snake_case)  (TLDR: underscores to separate words and all lowercase)

Here's a [good blog post](https://jeroenmols.com/blog/2016/03/07/resourcenaming/) on the topic of xml naming & conventions in android.